src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx

```diff
@@ -27,3 +123,44 @@ export const MyCollectionArtworkSWASectionSubmitted = () => {
     </>
   )
 }
+
+interface SubmissionStatusModalProps {
+  show: boolean
+  onClose(): void
+}
+
+const SubmissionStatusModal: React.FC<SubmissionStatusModalProps> = ({
+  show,
+  onClose,
+}) => {
+  if (!show) return null
+
+  return (
+    <ModalDialog
+      onClose={onClose}
+      title="Submission Status"
+      footer={
+        <Button onClick={onClose} width="100%">
+          Close
+        </Button>
+      }
+    >
+      <Text variant="sm-display">What does my Artwork’s status mean?</Text>
+      <Text as="li" variant="sm-display">
+        Submission in Progress - the artwork is being reviewed or is in the
+        sales process.
+      </Text>
+      <Text as="li" variant="sm-display" mt={2}>
+        Evaluation Complete - our specialists have reviewed this submission and
+        determined that we do not currently have a market for it.
+      </Text>
+      <Text variant="sm-display" mt={2}>
+        Have a question?
+      </Text>
+      <Text variant="sm-display">
+        Visit our help center or get in touch with one of our specialists at
+        sell@artsymail.com.
```

Comment:
**suggestion:** maybe we can make this email link clickable

Reply:
And it should be sell@artsy.net 👍

Reply:
Thanks Daria